tests/assets.rs

```diff
@@ -31,8 +32,29 @@ fn all_themes_are_present() {
             "base16",
             "base16-256",
             "gruvbox",
+            "gruvbox-dark",
+            "gruvbox-dark-hard",
+            "gruvbox-dark-hard-ndc",
+            "gruvbox-dark-medium",
+            "gruvbox-dark-medium-ndc",
+            "gruvbox-dark-ndc",
+            "gruvbox-dark-soft",
+            "gruvbox-dark-soft-ndc",
+            "gruvbox-hard",
+            "gruvbox-hard-ndc",
             "gruvbox-light",
-            "gruvbox-white",
+            "gruvbox-light-hard",
+            "gruvbox-light-hard-ndc",
+            "gruvbox-light-medium",
+            "gruvbox-light-medium-ndc",
+            "gruvbox-light-ndc",
+            "gruvbox-light-soft",
+            "gruvbox-light-soft-ndc",
+            "gruvbox-medium",
+            "gruvbox-medium-ndc",
+            "gruvbox-ndc",
+            "gruvbox-soft",
+            "gruvbox-soft-ndc",
```

Comment:
Phew. That's an awful lot of different themes. To be honest, I would really prefer if they are not all part of `bat`. Light/dark variants are fine, but 23 different variations?!

Reply:
Yes, it's a lot. Thought you could ask. There are 2 ways:
1. Removing some of them (e.g. `ndc` ones, there are not a part of official gruvbox)
2. Changing the test to check if there are some themes without checking for other ones (assets could have some unchecked themes)
Also 6 of them are symlinks

Reply:
> Changing the test to check if there are some themes without checking for other ones (assets could have some unchecked themes)

Well, I don't care too much about how the test code looks like, but if we add 23 gruvbox themes,  the `bat --list-themes` output will be completely overwhelmed.

Reply:
That's true. I believe `gruvbox`, `gruvbox-dark` and `gruvbox-light` is enough.

Reply:
Either this pr is invalid and bat should keep three variants or bat should learn how to work with theme variants

Reply:
@subnut?

Reply:
> Removing some of them (e.g. ndc ones, there are not a part of official gruvbox)

I am perfectly fine with it. Actually, I also had thought _what's the NDC for?_ but was unable to find anything, so I just included them in the repo anyway......

@j0hnmeow I think we should remove the NDC ones from the repo. Please push a commit at subnut/gruvbox-tmTheme#1 that removes all the NDC tmThemes

<br>

> Either this pr is invalid and bat should keep three variants or bat should learn how to work with theme variants

I think I will go with _bat should learn how to work with theme variants_ :smile: 

@sharkdp Just a suggestion for the future.... I can only see requests for new themes to be included in `bat` to keep growing as bat becomes more popular. So, I think, at some point, you should port the themes to some separate repo, say `bat-themes`, (yes, I am taking references from `vim-airline`) and include some mechanism to auto-build the theme cache when `bat` is updated. (Currently we need to manually re-build the cache ourselves) (This could be done by keeping all the tmThemes in a folder specified in the config file, so that when bat detects that the cache is invalid, it automatically re-builds the cache by using all the tmTheme files from that directory)

<br>

> if we add 23 gruvbox themes, the bat --list-themes output will be completely overwhelmed

I have a suggestion... Make `bat --list-themes` do just that, i.e. **list** the themes
Then the user could use a new `--preview-themes` to see the _preview_ of the themes 

Or, alternatively, implement `--preview-theme <theme_name>` to see the preview of <theme_name>, and show preview of all themes when `--preview-all-themes`

Reply:
> > if we add 23 gruvbox themes, the bat --list-themes output will be completely overwhelmed
> 
> I have a suggestion... Make `bat --list-themes` do just that, i.e. **list** the themes
> Then the user could use a new `--preview-themes` to see the _preview_ of the themes
> 
> Or, alternatively, implement `--preview-theme <theme_name>` to see the preview of <theme_name>, and show preview of all themes when `--preview-all-themes`

There's something similar in [dandavison/delta](https://github.com/dandavison/delta)